static/css/homepage.css

```diff
@@ -0,0 +1,55 @@
+.bio-wrapper {
+  background-image: url('/img/bio_bg.png');
+  background-size: cover;
+  background-position: center;
+}
+
+.bio_title, .project_title, .recentPost_title {
+  font-size: 50px;
+  font-family: gabriola;
```

Comment:
I dont understand why you change the font to "gabriola", I never asked for this change